solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRScoringQuery.java

```diff
@@ -73,6 +74,8 @@
   final private Map<String,String[]> efi;
   // Original solr query used to fetch matching documents
   private Query originalQuery;
+  // Model was picked for this Docs
+  private Set<Integer> pickedInterleavingDocIds;
```

Comment:
4.1/n The addition of a new member here which is only sometimes applicable got me wondering about possibly having `LTRInterleavingScoringQuery extends LTRScoringQuery` inheritance.